### ISSUE TITLE
TypeError: Cannot read property 'forEach' of undefined. Episode 03

### ISSUE THREAD

**Issue by CT-Shadow** [The-SourceCode/Discord.js-Bot-Development#44]

**Comment by ItzWillie** [comment]
I agree. I looked over the code many times, but I can't find any problems except for that...

**Comment by CT-Shadow** [comment]
Yes, it is very strange...

**Comment by PantheraRed** [comment]
Look for the [`array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) before [`.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) which is being declared as an `undefined` object. The `array` before `.forEach` must be defined.

**Comment by MathusanP** [comment]
.forEach can only be used in Arrays

**Comment by PantheraRed** [comment]
Isn't that what I said 🐢
